pysingfel/particle.py

```diff
@@ -459,12 +459,12 @@ def create_masks(self):
         self.solute_mask  = self.create_solute_mask(dry=True)
         self.solvent_mask = self.solute_mask * ~self.create_solute_mask(dry=False)
 
-    def create_other_mask(self, virus_void=False):
+    def create_other_mask(self, virus_void=False, probe_scale=0.5):
```

Comment:
I wonder whether probe_scale should be initialized as self.probe_scale in this method

Reply:
Consider updating examples/notebooks/virus_filling.ipynb by explicitly passing probe_scale=0.5 in the example. 
So that people using the notebook is aware this is something that can be set

Reply:
I will update the notebook soon after the numba/CUDA issue is resolved on psana.

Reply:
I've merged this PR now. Let's update the notebook in a different PR

Reply:
OK, will do that soon

Reply:
@chuckie82 I took care of this in the "cupy" PR for Skopi